doc/conf.py

```diff
@@ -52,6 +52,17 @@
     raise ImportError("No module named numpydoc - you need to install "
                       "numpydoc to build the documentation.")
 
+try:
+    import colorspacious
+except ImportError:
+    raise ImportError("No module named colorspacious - you need to install "
+                      "colorspacious to build the documentation")
+
+try:
+    import mock
```

Comment:
`mock` is only required on Python 2.7; maybe this test should be merged with the import on line 291?